acctest/plugin/plugin_acc_test.go

```diff
@@ -69,27 +67,31 @@ func TestAccInitAndBuildBasicAmazonEbs(t *testing.T) {
 	acctest.TestPlugin(t, testCase)
 }
 
-func cleanupPluginInstallation(plugin addrs.Plugin) error {
+func pluginDirectory(plugin addrs.Plugin) (string, error) {
 	home, err := homedir.Dir()
 	if err != nil {
-		return err
+		return "", err
 	}
-	pluginPath := filepath.Join(home,
+	if xdgConfigHome := os.Getenv("XDG_CONFIG_HOME"); xdgConfigHome != "" {
```

Comment:
Yep, that's essentially what `pathing.configDir` does (for UNIX systems at least).
That code I changed since it did not support plugins with a depth !=3, but truth be told rather than trying to figure ourselves what the expected directory is supposed to be here (with OS-specific code), we can lean on `packer.PluginFolder`